src/WebJobs.Script.WebHost/Web.config

```diff
@@ -148,7 +148,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.IO.Compression" publicKeyToken="b77a5c561934e089" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.0.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-4.1.2.0" newVersion="4.1.2.0" />

```

Comment:
Need to undo this change. We need to keep the redirect pushing it back to the built in version (4.0.0)